code/obj/machinery/nuclearbomb.dm

```diff
@@ -249,7 +250,10 @@
 			playsound(src.loc, 'sound/impact_sounds/Metal_Hit_Light_1.ogg', 100, 1)
 			attack_particle(user,src)
 
-		..()
+		if (istype(W, /obj/item/wrench/battle) && src.health <= src.max_health)
```

Comment:
Consider location relative to `src.take_damage()`

Reply:
AHA